python_modules/dagster/dagster/_cli/blueprint.py

```diff
@@ -0,0 +1,97 @@
+import json
+from importlib import import_module
+from pathlib import Path
+from typing import Sequence
+
+import click
+
+from dagster import _check as check
+from dagster._core.blueprints.load_from_yaml import YamlBlueprintsLoader
+from dagster._core.definitions.load_assets_from_modules import find_objects_in_module_of_types
+
+
+@click.group(name="blueprint")
+def blueprint_cli():
+    """Commands for working with Dagster blueprints."""
+
+
+@blueprint_cli.command()
+@click.option(
+    "--loader-module",
+    type=click.STRING,
+    help="Path of Python module that contains YamlBlueprintsLoader objects",
+    required=True,
+)
+@click.option(
+    "--vscode-folder-path",
+    type=click.STRING,
+    help="Path to the vscode workspace folder.",
```

Comment:
We could probably infer both of these so you could just drop a `dagster blueprint configure-vscode` - the vscode workspace by looking for a `.vscode` directory and python module from your `pyproject.toml`

Reply:
+1, I think it would be nice to walk up the directory tree looking for `.vscode`